### PR TITLE
Add Chainstack to BSC node providers

### PR DESCRIPTION
**Pull request by akegaviar**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ In the below sections you can find a list of different layers of the BSC Stack.
 ### :factory: Infrastructure
 | Components | Existing projects | Potentially interesting projects
 |-|-|-
-|API/Node access| [Ankr](https://www.ankr.com/),[NowNodes](https://nownodes.io/blog/binance-smart-chain-an-introduction),[QuikNode](https://www.quiknode.io/)
+|API/Node access| [Chainstack](https://chainstack.com/protocols/binance-smart-chain/),[Ankr](https://www.ankr.com/),[NowNodes](https://nownodes.io/blog/binance-smart-chain-an-introduction),[QuikNode](https://www.quiknode.io/)
```
Comment (easeev) [inline Readme.md:34]:
```suggestion
|API/Node access| [Ankr](https://www.ankr.com/),[Chainstack](https://chainstack.com/build-better-with-binance-smart-chain/),[NowNodes](https://nownodes.io/blog/binance-smart-chain-an-introduction),[QuikNode](https://www.quiknode.io/)
```

1. Ordered providers in alphabetical order
2. Changed link for Chainstack

**Review by easeev — CHANGES_REQUESTED** [review]

File: Readme.md [hunk 2]
```diff
@@ -31,7 +31,7 @@ In the below sections you can find a list of different layers of the BSC Stack.
 ### :factory: Infrastructure
 | Components | Existing projects | Potentially interesting projects
 |-|-|-
-|API/Node access| [Ankr](https://www.ankr.com/),[NowNodes](https://nownodes.io/blog/binance-smart-chain-an-introduction),[QuikNode](https://www.quiknode.io/)
+|API/Node access| [Ankr](https://www.ankr.com/),[Chainstack](https://chainstack.com/build-better-with-binance-smart-chain/),[NowNodes](https://nownodes.io/blog/binance-smart-chain-an-introduction),[QuikNode](https://www.quiknode.io/)
 |Archive Node Service| [InfStones](https://infstones.com/)
```
Comment (easeev) [inline Readme.md:35]: Add Chainstack as well

**Review by easeev — CHANGES_REQUESTED** [review]
LGTM

File: Readme.md (hunk 2 above)
Comment (akegaviar) [reply Readme.md]: Done

**Review by easeev — APPROVED** [review]
LGTM

**Pull request merged**